internal/store/certificatesigningrequest.go

```diff
@@ -38,7 +38,7 @@ var (
 			Type: metric.Gauge,
 			Help: descCSRLabelsHelp,
 			GenerateFunc: wrapCSRFunc(func(j *certv1beta1.CertificateSigningRequest) *metric.Family {
-				labelKeys, labelValues := kubeLabelsToPrometheusLabels(j.Labels)
+				labelKeys, labelValues := kubeMetadataToPrometheusLabels(labelPrefix, j.Labels)
```

Comment:
Nit but I think i would prefer these to be explicit functions for labels/annotations.

Reply:
I think if we rename it to something generic like `kubeMetadataToPrometheusMapping` or `kubeMetadataToPrometheusValue` or something along those lines, and add a comment to the function saying it can return either label or annotation, I think this would make sense IMHO.

Reply:
The prefix is used to prefix the label when it's converted to a prometheus label.

Reply:
I can understand that. If that is preferred (thoughts @andyxning ?), I can just  change this PR to rename the method `kubeAnnotationsToPrometheusAnnotations` to `kubeAnnotationsToPrometheusLabels`.

Reply:
> I can just change this PR to rename the method `kubeAnnotationsToPrometheusAnnotations` to `kubeAnnotationsToPrometheusLabels`.

Seems reasonable. I agree.

Reply:
Yep, I prefer that as well.